scalyr_agent/configuration.py

```diff
@@ -1600,6 +1600,13 @@ def enable_worker_process_metrics_gather(self):
         """
         return self.__get_config().get_bool("enable_worker_process_metrics_gather")
 
+    @property
+    def enable_copy_truncate_log_rotation(self):
```

Comment:
I would rename this to `enable_copy_truncate_log_rotation_support` since otherwise it sounds like we are using copy truncate to rotate our own logs.